packages/api/test/e2e/doughnut.e2e.ts

```diff
@@ -0,0 +1,210 @@
+// Copyright 2019 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import testingPairs from '@polkadot/keyring/testingPairs';
+import { hexToU8a, assert } from '@polkadot/util'
+import { CENNZnut as encodeCennznut} from '@cennznet/cennznut-wasm';
+import { Doughnut as DoughnutMaker } from '@plugnet/doughnut-wasm';
+// import { Doughnut as DoughnutMaker } from '/Users/karishma/WebstormProjects/doughnut-rs/js/libNode/Doughnut';
+import { Api } from '../../src/Api';
+import { Keypair } from '@plugnet/util-crypto/types';
+import {Keyring} from '@polkadot/api';
+import { KeyringPair } from '@plugnet/keyring/types';
+import initApiPromise from '../../../../jest/initApiPromise';
+import {cryptoWaitReady} from '@plugnet/util-crypto';
+import Doughnut from '@cennznet/types/Doughnut';
+
+// Helper for creating CENNZnuts
+function makeCennznut(module: string, method: string): Uint8Array {
+    const moduleVec = [
+        [
+            module,  {
+                        "name":module,
+                        "methods":[
+                            [
+                                method,  {
+                                    "name":method,
+                                    "constraints":null
+                                }
+                            ],
+                        ]
+                    }
+        ],
+    ];
+    const contractVec = [];
+    const cennznut = new encodeCennznut(moduleVec, contractVec);
+    return cennznut.encode();
+}
+
+/// Helper for creating v0 Doughnuts
```

Comment:
Would be good to mention any fixed/constant aspect of the created doughnut. Things like expiry, not before and signature method.